applications/directory/src/components/Link.js

```diff
@@ -4,7 +4,7 @@ import NextLink from 'next/link';
 import { Link as MuiLink } from '@mui/material';
 
 const Link = forwardRef(({ href, children, ...props }, ref) => {
-  const isExternal = href.match(/^https?:/);
+  const isExternal = href.match(/^https?:/); // TODO - make url that starts with www. external as well??
```

Comment:
Nope -- all links should always start with http or https. Links starting with just www are invalid. Good thinking though.

Reply:
got it

Reply:
I will take care of it in oksana branch